### ISSUE TITLE
Console output is badly formated

### ISSUE THREAD

**Issue by pakrym** [Microsoft/vstest#478]
Some lines are prepended with `Information: [xUnit.net 00:00:00.7500290]` others are not, also additional line breaks are added making it very hard to read.

```
>> dotnet --version
1.0.0-rc4-004771

>> dotnet test

Build started, please wait...
Build completed.

Test run for D:\github\aspnet\Testing\test\Microsoft.AspNetCore.Testing.Tests\bin\Debug\netcoreapp1.1\Microsoft.AspNetCore.Testing.Tests.dll(.NETCoreApp,Version=v1.1)
Microsoft (R) Test Execution Command Line Tool Version 15.0.0.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
Information: [xUnit.net 00:00:00.7500290]   Discovering: Microsoft.AspNetCore.Testing.Tests

Information: [xUnit.net 00:00:01.0069526]   Discovered:  Microsoft.AspNetCore.Testing.Tests

Information: [xUnit.net 00:00:01.0803985]   Starting:    Microsoft.AspNetCore.Testing.Tests

Warning: [xUnit.net 00:00:01.2145519]     Microsoft.AspNetCore.Testing.ConditionalFactTest.ConditionalFactSkip [SKIP]

Information: [xUnit.net 00:00:01.2152276]       Test is always skipped.

Warning: [xUnit.net 00:00:01.2372481]     Microsoft.AspNetCore.Testing.xunit.OSSkipConditionClassTest.TestSkipClassWindows [SKIP]

Information: [xUnit.net 00:00:01.2374496]       Test cannot run on this operating system.

Warning: [xUnit.net 00:00:01.2379220]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMac_TrueOnMac [SKIP]

Information: [xUnit.net 00:00:01.2380677]       Test cannot run on this operating system.

Skipped  Microsoft.AspNetCore.Testing.ConditionalFactTest.ConditionalFactSkip
Skipped  Microsoft.AspNetCore.Testing.xunit.OSSkipConditionClassTest.TestSkipClassWindows
Skipped  Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMac_TrueOnMac
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.TestSkipMacOSX
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.DoesNotSkip_WhenVersionsDoNotMatch
Passed   Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsWindows_TrueOnWindows
Passed   Microsoft.AspNetCore.Testing.ConditionalTheoryTest.ConditionalTheoryRunOncePerDataLine(arg: 0)
Passed   Microsoft.AspNetCore.Testing.ExceptionAssertTest.AssertArgumentNullOrEmptyString_WorksInNonEnglishCultures
Warning: [xUnit.net 00:00:01.3522855]     Microsoft.AspNetCore.Testing.ConditionalTheoryTest.ConditionalTheorySkip [SKIP]

Information: [xUnit.net 00:00:01.3523885]       Test is always skipped.

Warning: [xUnit.net 00:00:01.3527142]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMono_TrueOnMono [SKIP]

Information: [xUnit.net 00:00:01.3527755]       Test cannot run on this runtime framework.

Warning: [xUnit.net 00:00:01.3528896]     Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.TestSkipWindows [SKIP]

Information: [xUnit.net 00:00:01.3529382]       Test cannot run on this operating system.

Passed   Microsoft.AspNetCore.Testing.ConditionalTheoryTest.ConditionalTheoryRunOncePerDataLine(arg: 1)
Skipped  Microsoft.AspNetCore.Testing.ConditionalTheoryTest.ConditionalTheorySkip
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.TestSkipLinux
Passed   Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMono_FalseElsewhere
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.DoesNotSkip_WhenOnlyVersionsMatch
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.RunTest_DoesNotRunOnWin7OrWin2008R2
Skipped  Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMono_TrueOnMono
Skipped  Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.TestSkipWindows
Warning: [xUnit.net 00:00:01.3608452]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsLinux_TrueOnLinux [SKIP]

Information: [xUnit.net 00:00:01.3608985]       Test cannot run on this operating system.

Skipped  Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsLinux_TrueOnLinux
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.Skips_WhenVersionsMatches(currentOSVersion: "2.5", skipVersion: "2.5")
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.Skips_WhenVersionsMatches(currentOSVersion: "blue", skipVersion: "Blue")
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.Skips_WhenOnlyOperatingSystemIsSupplied
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.DoesNotSkip_WhenOperatingSystemDoesNotMatch
Passed   Microsoft.AspNetCore.Testing.ExceptionAssertTest.AssertArgumentOutOfRangeException_WorksInNonEnglishCultures
Passed   Microsoft.AspNetCore.Testing.xunit.OSSkipConditionAttributeTest.Skips_WhenVersionsMatchesOutOfMultiple
Passed   Microsoft.AspNetCore.Testing.RepalceCultureAttributeTest.DefaultsTo_EnGB_EnUS
Passed   Microsoft.AspNetCore.Testing.RepalceCultureAttributeTest.UsesSuppliedCultureAndUICulture
Information: [xUnit.net 00:00:01.3739585]   Finished:    Microsoft.AspNetCore.Testing.Tests

Passed   Microsoft.AspNetCore.Testing.RepalceCultureAttributeTest.BeforeAndAfterTest_ReplacesCulture
Passed   Microsoft.AspNetCore.Testing.TaskExtensionsTest.TimeoutAfterTest

Total tests: 27. Passed: 20. Failed: 0. Skipped: 7.
Test Run Successful.
Test execution time: 2.4516 Seconds


Test run for D:\github\aspnet\Testing\test\Microsoft.AspNetCore.Testing.Tests\bin\Debug\net451\Microsoft.AspNetCore.Testing.Tests.dll(.NETFramework,Version=v4.5.1)
Microsoft (R) Test Execution Command Line Tool Version 15.0.0.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
Warning: No test is available in D:\github\aspnet\Testing\test\Microsoft.AspNetCore.Testing.Tests\bin\Debug\net451\Microsoft.AspNetCore.Testing.Tests.dll. Make sure that installed test discoverers & executors, platform & framework version settings are appropriate and try again.


```

**Comment by codito** [comment]
Extra line breaks seem to be bug.

The lines prepended with `[Information]` and `[Warning]` come from the adapter itself. `Passed`, `Skipped` etc. are coming from console logger in dotnet-test. Clearly this is confusing, e.g. following information is duplicated:
```
Warning: [xUnit.net 00:00:01.3608452]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsLinux_TrueOnLinux [SKIP]

Information: [xUnit.net 00:00:01.3608985]       Test cannot run on this operating system.

Skipped  Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsLinux_TrueOnLinux
```

@pvlakshm @sbaid @singhsarab we should consider making `minimal` the default console verbosity. Related https://github.com/Microsoft/vstest/issues/301. Here's how the output looks when all Passed/Skipped info is trimmed:

```
Information: [xUnit.net 00:00:00.7500290]   Discovering: Microsoft.AspNetCore.Testing.Tests
Information: [xUnit.net 00:00:01.0069526]   Discovered:  Microsoft.AspNetCore.Testing.Tests
Information: [xUnit.net 00:00:01.0803985]   Starting:    Microsoft.AspNetCore.Testing.Tests
Warning: [xUnit.net 00:00:01.2145519]     Microsoft.AspNetCore.Testing.ConditionalFactTest.ConditionalFactSkip [SKIP]
Information: [xUnit.net 00:00:01.2152276]       Test is always skipped.
Warning: [xUnit.net 00:00:01.2372481]     Microsoft.AspNetCore.Testing.xunit.OSSkipConditionClassTest.TestSkipClassWindows [SKIP]
Information: [xUnit.net 00:00:01.2374496]       Test cannot run on this operating system.
Warning: [xUnit.net 00:00:01.2379220]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMac_TrueOnMac [SKIP]
Information: [xUnit.net 00:00:01.2380677]       Test cannot run on this operating system.
Warning: [xUnit.net 00:00:01.3522855]     Microsoft.AspNetCore.Testing.ConditionalTheoryTest.ConditionalTheorySkip [SKIP]
Information: [xUnit.net 00:00:01.3523885]       Test is always skipped.
Warning: [xUnit.net 00:00:01.3527142]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsMono_TrueOnMono [SKIP]
Information: [xUnit.net 00:00:01.3527755]       Test cannot run on this runtime framework.
Warning: [xUnit.net 00:00:01.3528896]     Microsoft.AspNetCore.Testing.xunit.OSSkipConditionTest.TestSkipWindows [SKIP]
Information: [xUnit.net 00:00:01.3529382]       Test cannot run on this operating system.
Warning: [xUnit.net 00:00:01.3608452]     Microsoft.AspNetCore.Testing.TestPlatformHelperTest.IsLinux_TrueOnLinux [SKIP]
Information: [xUnit.net 00:00:01.3608985]       Test cannot run on this operating system.
Information: [xUnit.net 00:00:01.3739585]   Finished:    Microsoft.AspNetCore.Testing.Tests

Total tests: 27. Passed: 20. Failed: 0. Skipped: 7.
Test Run Successful.
Test execution time: 2.4516 Seconds
```

It will be awesome if we can remove `Information`, `Warning` and replace that with color coding: Warning - Yellow, Information - Default Foreground.

**Comment by codito** [comment]
Check the behavior in dotnet-test-xunit? User should not get an impression that test run is hung.

Open: should SKIP be not shown in minimal output.

**Comment by onovotny** [comment]
Xunit has hung test detection and will send info to the diagnostics if there is one.

**Comment by codito** [comment]
Fixed with 34f59d52362ea5aa2769439c6c0eb0d018878d92 in 15.0-rtm and #524 for master.

**Comment by codito** [comment]
Fix available in dotnet-cli `.NET Command Line Tools (1.0.0-rc4-004854)`.
